src/js/App/Sidenav/Navigation/ChromeLink.js

```diff
@@ -60,8 +60,9 @@ const LinkWrapper = ({ href, isBeta, onLinkClick, className, currAppId, appId, c
 
     dispatch(appNavClick({ id: actionId }, domEvent));
   };
+  const quickStartHighlightId = href.split('/').slice(-1);
```

Comment:
I am thinking that maybe we should just use the full `href` string to avoid duplicate ids.

Reply:
i can't use `/` in the highlight id so i turn them into underscores